src/main/java/uk/gov/companieshouse/itemhandler/kafka/EmailSendKafkaProducer.java

```diff
@@ -12,13 +10,11 @@
 import uk.gov.companieshouse.kafka.producer.Acks;
 import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
 import uk.gov.companieshouse.kafka.producer.ProducerConfig;
-import uk.gov.companieshouse.logging.Logger;
-import uk.gov.companieshouse.logging.LoggerFactory;
 
 
 @Service
 public class EmailSendKafkaProducer implements InitializingBean {
-    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAMESPACE);
+    
```

Comment:
I suppose we could have `private static final Logger LOGGER = LoggingUtils.getLogger();` to keep the brevity?